target/linux/ar71xx/base-files/etc/diag.sh

```diff
@@ -126,6 +126,9 @@ get_status_led() {
 	oolite)
 		status_led="$board:red:system"
 		;;
+	dw33d)
+		status_led="dw33d:blue:status"
+		;;
```

Comment:
Please use:

```
status_led="$board:blue:status"
```

This change allows to merge multiple identical status led definitions at a later point.